Automatization

FedotovaVera opened the pull request.

FedotovaVera committed:
Added html semantic structure

FedotovaVera committed:
Added html semantic structure2

FedotovaVera committed:
Correction

FedotovaVera committed:
Fonts - home work

FedotovaVera committed:
added local fonts

FedotovaVera committed:
first commit

FedotovaVera committed:
correct 2

FedotovaVera committed:
new commit

FedotovaVera committed:
new commit 0107

FedotovaVera committed:
the interval

FedotovaVera committed:
ok

FedotovaVera committed:
Delete .gitignore

FedotovaVera committed:
Delete Project_Default.xml

FedotovaVera committed:
Delete profiles_settings.xml

FedotovaVera committed:
Delete misc.xml

FedotovaVera committed:
Delete modules.xml

FedotovaVera committed:
Delete otus-css-fedotova.iml

FedotovaVera committed:
Delete vcs.xml

FedotovaVera committed:
new file:   .gitignore
	new file:   dist/index.html
	new file:   dist/main.js
	new file:   package-lock.json
	new file:   package.json
	renamed:    fonts/Roboto/Roboto-Black.ttf -> src/fonts/Roboto/Roboto-Black.ttf
	renamed:    fonts/Roboto/Roboto-BlackItalic.ttf -> src/fonts/Roboto/Roboto-BlackItalic.ttf
	renamed:    fonts/Roboto/Roboto-Bold.ttf -> src/fonts/Roboto/Roboto-Bold.ttf
	renamed:    fonts/Roboto/Roboto-BoldItalic.ttf -> src/fonts/Roboto/Roboto-BoldItalic.ttf
	renamed:    fonts/Roboto/Roboto-Italic.ttf -> src/fonts/Roboto/Roboto-Italic.ttf
	renamed:    fonts/Roboto/Roboto-Light.ttf -> src/fonts/Roboto/Roboto-Light.ttf
	renamed:    fonts/Roboto/Roboto-LightItalic.ttf -> src/fonts/Roboto/Roboto-LightItalic.ttf
	renamed:    fonts/Roboto/Roboto-Medium.ttf -> src/fonts/Roboto/Roboto-Medium.ttf
	renamed:    fonts/Roboto/Roboto-MediumItalic.ttf -> src/fonts/Roboto/Roboto-MediumItalic.ttf
	renamed:    fonts/Roboto/Roboto-Regular.ttf -> src/fonts/Roboto/Roboto-Regular.ttf
	renamed:    fonts/Roboto/Roboto-Thin.ttf -> src/fonts/Roboto/Roboto-Thin.ttf
	renamed:    fonts/Roboto/Roboto-ThinItalic.ttf -> src/fonts/Roboto/Roboto-ThinItalic.ttf
	renamed:    images/Header.svg -> src/images/Header.svg
	renamed:    images/Line 3.svg -> src/images/Line 3.svg
	renamed:    images/Line.jpg -> src/images/Line.jpg
	renamed:    images/Rectangle 5.svg -> src/images/Rectangle 5.svg
	renamed:    images/avery-klein-JaXs8Tk5Iww-unsplash 2.jpg -> src/images/avery-klein-JaXs8Tk5Iww-unsplash 2.jpg
	renamed:    images/avery-klein.jpg -> src/images/avery-klein.jpg
	renamed:    images/dawid-liberadzki-vPFbVB7DPZI-unsplash 1.jpg -> src/images/dawid-liberadzki-vPFbVB7DPZI-unsplash 1.jpg
	renamed:    images/dawid-liberadzki.jpg -> src/images/dawid-liberadzki.jpg
	renamed:    images/inside-weather-Uxqlfigh6oE-unsplash 1.jpg -> src/images/inside-weather-Uxqlfigh6oE-unsplash 1.jpg
	renamed:    images/inside-weather-mob.jpg -> src/images/inside-weather-mob.jpg
	renamed:    images/inside-weather.jpg -> src/images/inside-weather.jpg
	renamed:    images/jared-weiss-P3cB0ynqYBw-unsplash 1.jpg -> src/images/jared-weiss-P3cB0ynqYBw-unsplash 1.jpg
	renamed:    images/jared-weiss-mob.png -> src/images/jared-weiss-mob.png
	renamed:    images/jared-weiss.jpg -> src/images/jared-weiss.jpg
	renamed:    images/keghan-crossland-ZZxmc66SjfM-unsplash 1.jpg -> src/images/keghan-crossland-ZZxmc66SjfM-unsplash 1.jpg
	renamed:    images/keghan-crossland.jpg -> src/images/keghan-crossland.jpg
	renamed:    images/menu.svg -> src/images/menu.svg
	renamed:    images/sinitta-leunen-YaReHIzNdRw-unsplash 1.jpg -> src/images/sinitta-leunen-YaReHIzNdRw-unsplash 1.jpg
	renamed:    images/sinitta-leunen-mob.png -> src/images/sinitta-leunen-mob.png
	renamed:    images/sinitta-leunen.jpg -> src/images/sinitta-leunen.jpg
	renamed:    images/spurwing-agency-mob.png -> src/images/spurwing-agency-mob.png
	renamed:    images/spurwing-agency-r6LceZ3ckpM-unsplash 1.jpg -> src/images/spurwing-agency-r6LceZ3ckpM-unsplash 1.jpg
	renamed:    images/spurwing-agency.jpg -> src/images/spurwing-agency.jpg
	renamed:    images/stil-P1jQzyvqfns-unsplash 1.jpg -> src/images/stil-P1jQzyvqfns-unsplash 1.jpg
	renamed:    images/stil-_mvSe9q8_n4-unsplash 1.jpg -> src/images/stil-_mvSe9q8_n4-unsplash 1.jpg
	renamed:    images/stil.jpg -> src/images/stil.jpg
	renamed:    images/stil2.jpg -> src/images/stil2.jpg
	renamed:    index.html -> src/index.html
	new file:   src/index.js
	renamed:    style/fonts.css -> src/style/fonts.css
	renamed:    style/media.css -> src/style/media.css
	renamed:    style/style.css -> src/style/style.css
	new file:   webpack.config.js

alinagaripova commented:
в ваших ошибках говорится о том что у вас некореектно проставлен путь к изображениям и стилям
задайте корректный, пример как на скринах ниже
<img width="269" alt="image" src="https://github.com/FedotovaVera/otus-css-fedotova/assets/46562364/74832a52-104c-44aa-90bb-02f865877438">
<img width="355" alt="image" src="https://github.com/FedotovaVera/otus-css-fedotova/assets/46562364/37802dce-3323-49fd-9e98-e809e77d0f14">

FedotovaVera committed:
new file:   dist/05293339b79c89bafb22.css
	new file:   dist/43a638c6ce50f2b730cc.css
	new file:   dist/72038951a562ea7dd9fb.css
	new file:   dist/images/avery-klein-c46b94798d6a4e013396.jpg
	new file:   dist/images/dawid-liberadzki-290b805275f9c2c12fca.jpg
	new file:   dist/images/jared-weiss-9250683ae1c817a55b5c.jpg
	new file:   dist/images/jared-weiss-mob-ebbae31c0c3448dc2cbc.png
	new file:   dist/images/keghan-crossland-996555ac89fd6ecd8715.jpg
	new file:   dist/images/menu-0d1e890b265f8c47d658.svg
	new file:   dist/images/sinitta-leunen-b89c35228d067b92c694.jpg
	new file:   dist/images/sinitta-leunen-mob-f20381e9b7ff000b3cc5.png
	new file:   dist/images/spurwing-agency-452e17182f77c7c989a3.jpg
	new file:   dist/images/spurwing-agency-mob-63e6fbbd172307189862.png
	new file:   dist/images/stil-37486e9eaaff3ca550df.jpg
	new file:   dist/images/stil2-9f87dc9c7e42d295e6ae.jpg
	modified:   dist/index.html
	modified:   src/index.html

FedotovaVera committed:
1

FedotovaVera committed:
modified:   package-lock.json
	modified:   package.json
	deleted:    src/images/Line 3.svg
	deleted:    src/images/Line.jpg
	deleted:    src/images/Rectangle 5.svg
	modified:   src/images/avery-klein.jpg
	modified:   src/images/dawid-liberadzki.jpg
	modified:   src/images/inside-weather-mob.jpg
	modified:   src/images/inside-weather.jpg
	new file:   src/images/jared-weiss-mob.jpg
	modified:   src/images/jared-weiss.jpg
	modified:   src/images/keghan-crossland.jpg
	modified:   src/images/sinitta-leunen.jpg
	new file:   src/images/spurwing-agency-mob.jpg
	modified:   src/images/spurwing-agency.jpg
	modified:   src/images/stil.jpg
	modified:   src/images/stil2.jpg
	modified:   src/index.html
	modified:   src/style/media.css
	modified:   src/style/style.css

FedotovaVera committed:
Create webpack.yml

FedotovaVera committed:
Update webpack.yml

FedotovaVera committed:
Update package-lock.json

FedotovaVera committed:
Update webpack.yml

FedotovaVera committed:
Update package.json

FedotovaVera committed:
Update package-lock.json

FedotovaVera committed:
Update webpack.yml

FedotovaVera committed:
Update package-lock.json

FedotovaVera committed:
Update package-lock.json

FedotovaVera committed:
Update package.json

FedotovaVera committed:
Update webpack.config.js

FedotovaVera committed:
Update webpack.config.js

FedotovaVera committed:
Update webpack.yml

FedotovaVera committed:
Update index.html

FedotovaVera committed:
Update index.html

FedotovaVera committed:
1

FedotovaVera committed:
Update webpack.yml

FedotovaVera committed:
modified:   package.json

FedotovaVera committed:
modified:   .github/workflows/webpack.yml

FedotovaVera committed:
new file:   .browserslistrc
	modified:   webpack.config.prod.js

FedotovaVera committed:
modified:   package.json

FedotovaVera committed:
Update index.html

FedotovaVera committed:
Update package.json

FedotovaVera committed:
renamed:    src/style/fonts.css -> src/style/fonts.scss
	renamed:    src/style/media.css -> src/style/media.scss
	renamed:    src/style/style.css -> src/style/style.scss

FedotovaVera committed:
Merge branch 'automatization' of https://github.com/FedotovaVera/otus-css-fedotova into automatization

FedotovaVera committed:
Update package.json

FedotovaVera committed:
modified:   src/index.js

FedotovaVera committed:
Merge branch 'automatization' of https://github.com/FedotovaVera/otus-css-fedotova into automatization

FedotovaVera committed:
modified:   src/index.js

FedotovaVera committed:
modified:   src/style/fonts.scss

FedotovaVera committed:
modified:   src/index.js

FedotovaVera committed:
Update style.scss

FedotovaVera committed:
Update style.scss

FedotovaVera committed:
Update index.js

FedotovaVera committed:
1

FedotovaVera committed:
Merge remote-tracking branch 'origin/automatization' into automatization
